pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
```

Comment:
Adding this minimal pytest config so code editors can auto-discover tests for this project.